### PR TITLE
Fucking remove mindswap book recoil

### PR DESCRIPTION
**Pull request by ShiftyRail**
[loose][bugfix][featureloss]

**Literally crashes clients** to do its thing, send them a message that they're multikeying, messes with their spells and antag roles...

### CODE REVIEW

**Comment by Kurfursten** [comment]
Why don't we make mindswapping an actual proc of its own (like transfer_to is), and then have both the mindswap spell and this spellbook use that?

**Comment by ShiftyRail** [comment]
I don't know how any of this works and maybe it's up for a future PR. For now, it's just something that crashes clients when used, and also doesn't transfer your spells.
It doesn't work at all and should be redone from scratch.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -445,6 +445,10 @@
 	name = "spellbook of [spellname]" //Note, desc doesn't change by design
 	..()
 
+/obj/item/weapon/spellbook/oneuse/mindswap/recoil(var/mob/user)
+	qdel(src)
+
+/*
```
Comment (Exxion) [inline code/game/gamemodes/wizard/spellbook.dm]: Commenting out code with block comments fucks with version control
That's how we got MEGAMADCRAB

Comment (ShiftyRail) [reply code/game/gamemodes/wizard/spellbook.dm]: Fine

**Comment by DrSnips** [comment]
Could we at least get a comedic placeholder effect like the user vomits up a brain similar to that one virus effect?

**Comment by Blithering** [comment]
Teleport the book to their head cavity and have (a copy of) their brain fall out as @DrSnips says

**Comment by DrSnips** [comment]
I don't think removing their actual brain is a good idea and Im sure that won't cause any issues down the road. Just have it spawn a brain so it doesn't cause any organ gas.

**Pull request merged**